sktime/datasets/_data_io.py

```diff
@@ -154,22 +154,23 @@ def _load_dataset(name, split, return_X_y, return_type=None, extract_path=None):
     return_X_y: bool, optional (default=True)
```

Comment:
```suggestion
    return_X_y: bool, optional (default=True)
```

There is no default? (it is not true)

Reply:
good spot, fixed